src/model/BridgeGame.js

```diff
@@ -0,0 +1,70 @@
+const Validation = require('../utils/Validation');
+const {
+  ONE_TIME,
+  UP,
+  DOWN,
+  CROSSED,
+  BLOCKED,
+  BLANK,
+  ZERO,
+} = require('../utils/constants');
+
+class BridgeGame {
+  #retryingCount = ONE_TIME;
+  #upperBridge = [];
+  #lowerBridge = [];
+
+  static validateDirection(direction) {
+    Validation.checkBlank(direction);
+    Validation.checkStringType(direction);
+    Validation.checkUpperCaseOfDirection(direction);
+    Validation.checkValidDirection(direction);
+  }
+
+  canMove(direction, winningBridge) {
+    return winningBridge.isSameDirection(direction, this.#upperBridge.length);
+  }
+
+  move(direction, CAN_MOVE) {
+    if (direction === UP) {
+      this.#upperBridge.push(CAN_MOVE ? CROSSED : BLOCKED);
+      this.#lowerBridge.push(BLANK);
+    }
+
+    if (direction === DOWN) {
+      this.#lowerBridge.push(CAN_MOVE ? CROSSED : BLOCKED);
+      this.#upperBridge.push(BLANK);
+    }
```

Comment:
```suggestion
    if (direction === UP) {
      this.#upperBridge.push(CAN_MOVE ? CROSSED : BLOCKED);
      this.#lowerBridge.push(BLANK);
      return;
    }
    
    this.#lowerBridge.push(CAN_MOVE ? CROSSED : BLOCKED);
    this.#upperBridge.push(BLANK);
```
앞선 `if` 문에서 명시적으로 `return`을 해준다면 뒤따르는 `if` 문을 생략할 수 있을 것 같습니다..!

Reply:
좋은 의견 감사드려요 😄👍